cmd/hypper/flags_test.go

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright The Helm Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+// import (
```

Comment:
why is this all commented out?

Reply:
Initially I added it for the header and possible reminder, but indeed it can be dropped out, as all flag tests happen in the respective command tests for now.

Removed in https://github.com/mattfarina/hypper/pull/37/commits/066c6cd04d4f335d98835983958624d7e52e2503.